<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaReservas</class>
 <widget class="QWidget" name="VentanaReservas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>333</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>333</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Reservas</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>333</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>333</width>
     <height>51</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:  rgb(255, 0, 0);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0,0%);
font: 20pt &quot;Tw Cen MT Condensed&quot;;
border: None;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Reservas&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/imagenes/imagenes/logo-udemedellin-footer.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>61</y>
     <width>333</width>
     <height>439</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>333</width>
     <height>439</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>333</width>
     <height>439</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>390</y>
      <width>333</width>
      <height>3</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>333</width>
      <height>0</height>
     </size>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="tbutton_reserva">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>370</y>
      <width>48</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>48</width>
      <height>33</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 0);
image: url(:/imagenes/imagenes/calendar-clock (1).png);
border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../imagenes/calendar-clock (1).png</normaloff>../imagenes/calendar-clock (1).png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="tbutton_usuario">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>400</y>
      <width>48</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>48</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>48</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0%);
image: url(:/imagenes/imagenes/user.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../imagenes/user.png</normaloff>../imagenes/user.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="tbutton_noticias">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>400</y>
      <width>48</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>48</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>48</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0%);
image: url(:/imagenes/imagenes/forma.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../imagenes/forma.png</normaloff>../imagenes/forma.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="tbutton_opinion">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>48</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>48</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>48</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0%);
image: url(:/imagenes/imagenes/envelope.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../imagenes/envelope.png</normaloff>../imagenes/envelope.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pbutton_reservadia">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>311</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{
	background-color: rgb(230, 230, 230);
	border-radius: 10px;


}

QPushButton:hover{
	
	
	background-color: rgb(145, 145, 145);
	border-radius: 10px;

}


</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>200</width>
      <height>3</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>1</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  rgb(238, 238, 0);
border: None;
</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_diaActual">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0,0%);
font: 14pt &quot;Tw Cen MT Condensed&quot;;
border: None;</string>
    </property>
    <property name="text">
     <string>Lunes</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_mesActual">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0,0%);
font: 14pt &quot;Tw Cen MT Condensed&quot;;
border: None;</string>
    </property>
    <property name="text">
     <string>Julio</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_fechaActual">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0,0%);
font: 30pt &quot;Tw Cen MT Condensed&quot;;
border: None;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#f90808;&quot;&gt;18&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pbutton_reservadiaSig">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>311</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{
	background-color: rgb(230, 230, 230);
	border-radius: 10px;


}

QPushButton:hover{
	
	
	background-color: rgb(145, 145, 145);
	border-radius: 10px;

}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_mesSiguiente">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0,0%);
font: 14pt &quot;Tw Cen MT Condensed&quot;;
border: None;</string>
    </property>
    <property name="text">
     <string>Julio</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_diaSiguiente">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0,0%);
font: 14pt &quot;Tw Cen MT Condensed&quot;;
border: None;</string>
    </property>
    <property name="text">
     <string>Martes</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_fechaSiguiente">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0,0%);
font: 30pt &quot;Tw Cen MT Condensed&quot;;
border: None;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#57b400;&quot;&gt;19&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>270</y>
      <width>200</width>
      <height>3</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>1</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  rgb(238, 238, 0);
border: None;
</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_recuerdoDiaActual">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>140</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0,0%);
font: 14pt &quot;Tw Cen MT Condensed&quot;;
border: None;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_recuerdoDiaSiguiente">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>280</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0,0%);
font: 14pt &quot;Tw Cen MT Condensed&quot;;
border: None;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
